source/agora/node/FullNode.d

```diff
@@ -1205,8 +1205,6 @@ public class FullNode : API
 
     private void pushBlock (const Block block) @trusted
     {
-        if (this.is_shutting_down)
-            return;
```

Comment:
Why not keep this anyway? We are shutting down so no need to continue.

Reply:
I thought as you do but I think it's not so big impact because it could do only one iteration even if it enters the `foreach` scope. But it's better to keep it.